cypress/e2e/1-getting-started/addItemFormView.cy.js

```diff
@@ -0,0 +1,57 @@
+describe('Craft Circle landing page view', () => {
+  beforeEach(() => {
+    // cy.intercept("craft-circle-be.herokuapp.com/graphql", {fixture: "userProfile.json"})
+    cy.visit("http://localhost:3000")
+  });
+
```

Comment:
Thank you for setting up our testing structure! I know we will add to cover remaining user flows once more functionalities are plugged in.